.github/workflows/main.yml

```diff
@@ -0,0 +1,87 @@
+name: CI-CD
+
+on:
+  push:
+    branches:
+    - stable
```

Comment:
I've never used github workflows before, so I might be missing something. But should this not be `master`?

Reply:
Absolutely!